internal/zinc-compile-core/src/main/scala/sbt/internal/inc/CompilerArguments.scala

```diff
@@ -82,7 +83,13 @@ final class CompilerArguments(
    * @param addLibrary Flag to return the Scala library.
    */
   def createBootClasspath(addLibrary: Boolean): String = {
-    val originalBoot = System.getProperty("sun.boot.class.path", "")
+    def findBoot: String = {
+      import scala.collection.JavaConverters._
+      System.getProperties.asScala.iterator.collectFirst {
+          case (k, v) if k.endsWith(".boot.class.path") => v
```

Comment:
Have you done this to avoid making it specific to `scala.boot.class.path`? Is there any other valuable system variable that users `.boot.class.path` as the suffix?

Reply:
The code is from scalac. I believe the intent is to support non OpenJDK based JVMs that don't use `sun.boot.classpath`.

https://bugs.eclipse.org/bugs/show_bug.cgi?id=188648#c6
http://mail.openjdk.java.net/pipermail/build-dev/2013-November/011103.html